converters/common.js

```diff
@@ -253,6 +278,11 @@ const TuyaDataPoints = {
     hyMode: 128,
     hyChildLock: 129,
     hyAlarm: 130,
+
```

Comment:
Replace this empty line with `// Silvercrest`

Reply:
Done